ansible/roles/atmo-mount-volume/tasks/main.yml

```diff
@@ -1,5 +1,10 @@
 ---
 
+- name: Set MOUNT_OPTS Ubuntu < 16
+  set_fact:
+    MOUNT_OPTS: 'defaults,nobootwait'
+  when: ansible_distribution == "Ubuntu" and ansible_distribution_major_version < "16"
```

Comment:
If a user passes MOUNT_OPTS they'll be ignored for these defaults if on ubuntu 16

Reply:
Ah yeah, sloppy merge on my part. Need to change it to match edwin's vars. I need to double-check if changing to `nobootwait` is even necessary

Reply:
No sweat, merging was nice. There are ways in git to create merge that is only symbolic, meaning the merge shows two ancestors but actually ignores all changes introduced in one of them. I probably would have done that here. but doesn't really matter.